src/grapheme.rs

```diff
@@ -452,6 +462,38 @@ impl GraphemeCursor {
         }
     }
 
+    #[inline]
+    fn handle_incb_consonant(&mut self, chunk: &str, chunk_start: usize) {
```

Comment:
issue: please add comments explaining what this code does (doc comments) and an inline ones explaiining how it's trying to achieve that